dist/s_all

```diff
@@ -131,7 +131,8 @@ for f in `find . -name ${t_pfx}\*`; do
         # s_evergree_validate, $FAILED_CMD can be parsed incorrectly and result in a failing
         # script being ignored. Use `[^\w]` to ensure we match on the full command name and
         # not just a substring
-        FAILED_CMD=$(echo "$COMMANDS" | grep "${LOCAL_NAME}[^\w]")
```

Comment:
@lukech this was the other change I was mentioning. The end result is we now properly print the name of the failing check rather than "". For example
```
####################### MESSAGE ############################
s_all run of: "" resulted in:
    ==== test/csuite/wt4156_metadata_salvage/main.c
    ASAN
#######################
```
now prints
```
####################### MESSAGE ############################
s_all run of: "./s_string" resulted in:
    ==== test/csuite/wt4156_metadata_salvage/main.c
    ASAN
#######################
```

Reply:
Oh nice, this is WT-10643. Thanks!

Reply:
No worries 👍. 
I had a quick skim of the remaining functions and I think misses the python cases:
```
2>&1 python3 function.py > ${t_pfx}py_function
2>&1 python3 style.py > ${t_pfx}py_style
2>&1 python3 comment_style.py ${fast} > ${t_pfx}py_comment_style
2>&1 python3 type_to_str.py > ${t_pfx}py_type_to_str"
```

I'll move this change to WT-10643 and pick that up next for a full fix.